app/helpers/script_helper.rb

```diff
@@ -1,6 +1,6 @@
 # rubocop:disable Rails/HelperInstanceVariable
 module ScriptHelper
-  include Webpacker::Helper
+  MANIFEST_RELATIVE_PATH = ['app', 'assets', 'builds', 'assets-manifest.json'].freeze
```

Comment:
I'd throw a `File.join` on here since it looks like we don't need the array for anything?
```suggestion
  MANIFEST_RELATIVE_PATH = File.join('app', 'assets', 'builds', 'assets-manifest.json').freeze
```

Reply:
Would that work if passed to `Rails.root.join` as we're using it for below?

Alternatively, would `Rails.root.join` be valid to use for the constant assignment?

Reply:
- Yup `Rails.root.join` is happy to take a path
- Yup we could just put that as the constant value too, I wasn't sure if you wanted to defer evaluation of `Rails.root.join` or something

Reply:
> * Yup we could just put that as the constant value too, I wasn't sure if you wanted to defer evaluation of `Rails.root.join` or something

Nope, just wasn't sure if `Rails.root.join` was fine to do in a constant assignment. I'll do that instead 👍

Reply:
Addressed in 03e73fa3ab7313829eca830362873eadb94b5968.